zio-schema/shared/src/main/scala/zio/schema/SchemaOrdering.scala

```diff
@@ -0,0 +1,87 @@
+package zio.schema
+
+import scala.annotation.tailrec
+
+import zio.Chunk
+import zio.schema.DynamicValue._
+import zio.schema.StandardType.UnitType
+
+object SchemaOrdering {
```

Comment:
I might make all of this package private since the user should only ever have to use `Schema#ordering` to get access to this, and we can avoid cluttering the Scaladoc API.

Reply:
this is updated now